apps/gallery/tests/test_apis.py

```diff
@@ -205,3 +209,38 @@ def save_file_with_api(self, kwargs={}):
         return response.data['id']
 
     # NOTE: Test for files
+
+
+class PrivateAttachmentFileViewTest(TestCase):
+    def setUp(self):
+        # Create a test user
+        self.user = User.objects.create_user(username='testuser', password='testpassword')
+
+        # Create a test entry attachment
+        self.project = Project.objects.create()
+        self.lead = Lead.objects.create(project=Project)
+        self.attachment = EntryAttachment.objects.create()
+        self.entry = Entry.objects.create(
+            lead=self.lead,
+            project=self.project,
+            entry=self.attachment
+        )
+        self.url = reverse('private_attachment_file_view', kwargs={
+            'module': PrivateFileModuleType.ENTRY_ATTACHMENT.value,
+            'identifier': urlsafe_base64_encode(force_bytes(self.attachment.id)),
+        })
+
+    def test_access_by_authenticated_user(self):
```

Comment:
To check
- Non-logged in -> 403?
- Normal User -> 403
- Member (N-C) -> 403
- Member (C) -> 200

For id > 1000000 -> all should be 403